### DISCUSSION TITLE
Is PPO tuner in HPO working for hyperparameter optimization?

### DISCUSSION THREAD

**Discussion by feufhd** [microsoft/nni#3758] [Q&A] [answered]
Hello! Thanks for your work.
I'm seaching for a HPO method based on PPO or other reinforcement learning models. I notice your PPO tuner under HPO, but it seems to only work for NAS, not hyperparameter optimization (like optimizing learning rate, batch size). Thank you!

**Comment by QuanluZhang** [comment, marked as answer]
@feufhd yes, for now, PPO tuner is specifically for NAS, but it can be ported to HPO scenarios. we will support it in future release. we also encourage you to contribute back if you write one before us :)

**Reply by scarlett2018** [reply]
Question been answered (no support yet). The new feature request (PPO Tuner for HPO) has been added to backlog https://github.com/microsoft/nni/discussions/3745.
